ISPN-12628 All REST response descriptions should contain possible HTTP response codes (#9040)
infinispan/infinispan: documentation/src/main/asciidoc/topics/ref_rest_cache_managers.adoc

POST /rest/v2/cache-managers/{cacheManagerName}/backups/{backupName}
----

> If a backup with the same name already exists, the service will responds  `409 (Conflict)`, if `directory` parameter has a wrong format, will responds `400 (Bad Request)`.

Reviewer: ```suggestion
If a backup with the same name already exists, the service responds with `409 (Conflict)`.
If the `directory` parameter is not valid, the service returns `400 (Bad Request)`.
```
